integration-tests/src/test/java/com/redhat/service/smartevents/integration/tests/steps/WebhookSiteSteps.java

```diff
@@ -26,27 +26,29 @@ public WebhookSiteSteps(TestContext context) {
         this.context = context;
     }
 
-    @Then("^Webhook site contains request with text \"(.*)\" within (\\d+) (?:minute|minutes)$")
-    public void webhookSiteContainsRequest(String requestText, int timeoutMinutes) {
+    @Then("^Webhook site with id \"(.*)\" contains request with text \"(.*)\" within (\\d+) (?:minute|minutes)$")
+    public void webhookSiteWithIdContainsRequest(String webhookID, String requestText, int timeoutMinutes) {
```

Comment:
```suggestion
    public void webhookSiteWithIdContainsRequest(String webhookId, String requestText, int timeoutMinutes) {
```
Also other occurrences below.